Update example config's MotD to our current MotD

nervere opened the pull request:
## Why it's good
This doesn't need to be done regularly, by any means, but the current one is literally a decade out of date and still says Baystation.

jwhitak commented:
I saw the old motd so many times this week while testing, and was so tempted to do this. Thank you!

SonixApache commented:
this kills the soul

pedr69 commented on config-example/motd.txt:
```diff
@@ -1,9 +1,14 @@
+-------- Info --------
+• <b>New player resources:</b> <a href="http://ss13.moe/wiki/index.php/The_Basics">The Basics</a> | <a href="http://ss13.moe/sekrit/rurus.html">Server Rules</a> | <a href="http://ss13.moe/wiki/index.php/Guides">Guides</a>
+• This server's code is <b>open-source</b>, running the Bleeding-Edge branch of /vg/ code. <a href="https://github.com/d3athrow/vgstation13">(Get it here)</a>
+• You can join our official Discord server <a href="https://discord.gg/RYnQZvxrmH">here.</a>
```
fuck off

nervere replied:
Great feedback, thank you!

nervere commented:
> this kills the soul

There is nothing "soulful" about an unmaintained codebase.

SonixApache commented:
> > this kills the soul
> 
> There is nothing "soulful" about an unmaintained codebase.

there is a lot soulful about an unmaintained codebase, actually

xXxEMBERGHOSTSQUADDIExXx commented:
remove discord

xXxEMBERGHOSTSQUADDIExXx commented:
and add forum link

nervere commented:
Forums link would be redundant, that's handled separately via the config and is accessible by a button in the top right.

Kurfursten commented:
I think there is a real objection about startup text inflation. How does this look on a standard window size?

nervere commented:
> I think there is a real objection about startup text inflation. How does this look on a standard window size?

Here's a screenshot from the live server, it's the exact same as what's in the PR.

![image](https://github.com/vgstation-coders/vgstation13/assets/26285377/ab4d3bc6-ea34-4938-a9ed-20041fdacef3)

Kurfursten commented:
Okay, size isn't too bad

nervere commented:
> The version in this PR seems to be slightly older than the version on live. Different BYOND build and different Pomf Facts. Also, I don't think the latter really needs to be in there.

Removed Pomf facts and fixed the version.

Eneocho commented:
Pomf facts bros we got too cocky...

Eneocho commented:
Pomf facts bros we got too cocky...
